code/modules/events/wisdomcow.dm

```diff
@@ -1,17 +1,60 @@
 /datum/round_event_control/wisdomcow
-	name = "Wisdom cow"
+	name = "Wisdom Cow"
 	typepath = /datum/round_event/wisdomcow
 	max_occurrences = 1
 	weight = 20
 	category = EVENT_CATEGORY_FRIENDLY
 	description = "A cow appears to tell you wise words."
+	admin_setup = list(
+		/datum/event_admin_setup/set_location/wisdom_cow,
+		/datum/event_admin_setup/listed_options/wisdom_cow,
+		/datum/event_admin_setup/input_number/wisdom_cow,
+	)
+
+/datum/round_event/wisdomcow
+	///Admin set overide location for the cow.
+	var/turf/admin_spawn_location
+	///Admin set override for the wisdom the cow will grant.
+	var/admin_selected_wisdom
+	///Admin set override for the amount of experience the wisdom cow will grant or remove
+	var/admin_selected_experience
```

Comment:
Instead of using `admin_foo` and `admin_bar`, just have the variables be called `spawn_location`, `skill`, `skill_experience_amount`, and then have it do the regular thing when they're null.

The fact that the current use of the vars would be for admin overrides doesn't preclude the (admitedly unlikely) that other parts of the codebase might want to trigger the wisdom cow event themselves.

(Hard to determine why they wouldn't just spawn the cow directly, unless they wanted to hook into the announcement message? 🤔 )